ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -189,25 +179,11 @@ Item {
                         return subItems
                     }
                     delegate: Loader {
-                        property bool isActiveChannel: chatRepeater.isChatActive(model.itemId)
-
                         id: categoryChatLoader
                         // Channels are not loaded by default and only load when first put active
-                        active: false
+                        active: model.active
                         width: parent.width
-                        height: isActiveChannel ? parent.height : 0
-
-                        Connections {
-                            id: loaderConnections
-                            target: categoryChatLoader
-                            // First time this channel turns active, activate the Loader
-                            onIsActiveChannelChanged: {
-                                if (categoryChatLoader.isActiveChannel) {
-                                    categoryChatLoader.active = true
-                                    loaderConnections.enabled = false
-                                }
-                            }
-                        }
```

Comment:
I would be careful removing all of this. I know it's ugly code, but it made it that channel switching was way faster and we kept the scroll position. 
Can you test that and revert if this broke it please. I have a task to refactor everything here in 0.10 anyway, so I'll clean it up once I'm there.

Reply:
@jrainville Thank you for the description. Fixed an issue with unloading